doc/development_process/release_process.rst

```diff
@@ -232,70 +232,94 @@ created.
-      $ git push git@github.com:zephyrproject-rtos/zephyr.git v1.11.0
+        #. Find the new tag at the top of the releases page and edit the release
+           with the ``Edit`` button with the following:
 
-      $ git tag -a zephyr-v1.11.0
-      <Use "Zephyr 1.11.0" as the tag annotation>
-      $ git push git@github.com:zephyrproject-rtos/zephyr.git zephyr-v1.11.0
+            * Name it ``Zephyr 1.11.0-rc1``
+            * Copy the shortlog into the release notes textbox (*don't forget
+              to quote it properly so it shows as unformatted text in Markdown*)
+            * Check the "This is a pre-release" checkbox
 
-#. If it's a release candidate, create a shortlog of changes between the
-   previous release::
+        #. Send an email to the mailing lists (``announce`` and ``devel``)
+           with a link to the release
 
-    $ git shortlog v1.10.0..v.1.11.0-rc1
+    .. tab:: Final Release
 
-#. Find the new tag at the top of the releases page, edit the release with the
-   ``Edit`` button and then do the following:
+        .. note::
 
-   * If it's a release candidate:
+            This section uses tagging 1.11.0 as an example, replace with the
+            appropriate final release version.
 
-     * Name it ``Zephyr 1.11.0-rc1``
-     * Copy the shortlog into the release notes textbox (don't forget to quote it
-       properly so it shows as unformatted text in Markdown)
-     * Check the "This is a pre-release" checkbox
-   * If it's a release:
+        When all final release criteria has been met and the final release notes
+        have been approved and merged into the repository, the final release version
+        will be set and repository tagged using the following procedure:
 
-     * Name it ``Zephyr 1.11.0``
-     * Copy the full content of ``docs/release-notes-1.11.rst`` into the the
-       release notes textbox
-     * Copy the full list of GitHub issues closed with this release into the
-       release notes textbox (see below on how to generate this list)
+        #. Update the version variables in the :zephyr_file:`VERSION` file
+           located in the root of the Git repository. Set ``EXTRAVERSION``
+           variable to zero to indicate final release::
 
-#. Send an email to the mailing lists (``announce`` and ``devel``) with a link
-   to the release
+            EXTRAVERSION = 0
+
+        #. Commit and post a PR of the updated :zephyr_file:`VERSION` file using
+           ``release: Zephyr 1.11.0`` as the commit subject. Verify that CI has
+           completed successfully.
+        #. Tag and push the version, using two annotated tags::
+
+            $ git tag -a v1.11.0
+            <Use "Zephyr 1.11.0" as the tag annotation>
+            $ git push git@github.com:zephyrproject-rtos/zephyr.git v1.11.0
+
+            $ git tag -a zephyr-v1.11.0
+            <Use "Zephyr 1.11.0" as the tag annotation>
+            $ git push git@github.com:zephyrproject-rtos/zephyr.git zephyr-v1.11.0
```

Comment:
Is it documented elsewhere why we need this second tag for a release? What's the rationale for it?

Reply:
This is needed to properly generate assets in https://github.com/zephyrproject-rtos/zephyr/releases, otherwise it is just a tarred version.

Reply:
@nashif, for clarity, is the simple tag needed for the assets? To @jhedberg point I'm thinking a comment/note somewhere, maybe here, as to why might help for understanding.

Reply:
we need both tags

Reply:
> @nashif, for clarity, is the simple tag needed for the assets?

no, the long one with zephyr-*